concepticondata/conceptlists/Speed-2017-485.tsv

```diff
@@ -0,0 +1,486 @@
+Speed-2017-485-455	455	wegverharding	road surfacing	0.333333333	0	0.533333333	2.133333333	3.266666667	zien	0.521276596	3.166666667	4.833333333	3.416666667		
+Speed-2017-485-456	456	wekker	alarm clock	4.333333333	0	0	1.133333333	2.066666667	horen	0.575221239	5.916666667	5.416666667	2.833333333		
+Speed-2017-485-457	457	whisky	whisky	0.133333333	3.733333333	3.666666667	0.133333333	2.8	proeven	0.343949045	3.583333333	5.166666667	3.583333333		
+Speed-2017-485-458	458	wiel	wheel	0.8	0	0	1.6	3.733333333	zien	0.608695652	4.916666667	4.416666667	4.583333333	710	WHEEL
+Speed-2017-485-459	459	wierook	incense	0	0.2	4.333333333	0.4	3	ruiken	0.546218487	3	4.416666667	4.333333333	3356	INCENSE
+Speed-2017-485-460	460	wijn	wine	0.8	4.266666667	3.4	0.866666667	3.533333333	proeven	0.269430052	3.416666667	4.583333333	4.666666667	1524	WINE
+Speed-2017-485-461	461	windgong	windgong	3.25	0	0	0.833333333	2.416666667	horen	0.5	3.166666667	3.916666667	4		
+Speed-2017-485-462	462	windmolen	windmill	2.25	0	0	0.875	4	zien	0.561403509	4.416666667	4.833333333	4.583333333		
+Speed-2017-485-463	463	witlof	chicory	0.333333333	4.066666667	2	1.133333333	3.8	proeven	0.329411765	3	4	3.083333333		
+Speed-2017-485-464	464	wol	wool	0.066666667	0	0.333333333	3.8	3.466666667	voelen	0.495652174	2.75	3.5	5.166666667	964	WOOL
+Speed-2017-485-465	465	wolf	wolf	2.8	0	0.066666667	0.933333333	4.133333333	zien	0.521008403	5	5.416666667	4	522	WOLF
+Speed-2017-485-466	466	wolk	cloud	0.933333333	0.066666667	0.4	1.533333333	4.133333333	zien	0.575471698	2.75	3.083333333	4.583333333	1489	CLOUD
+Speed-2017-485-467	467	wortel	carrot	0.8	4.466666667	1.2	1.933333333	3.933333333	proeven	0.297297297	3.75	3.75	4.75	3085	CARROT
+Speed-2017-485-468	468	zaagsel	sawdust	0.733333333	0.066666667	2.733333333	3.133333333	2.933333333	voelen	0.319444444	3	2.666666667	3.666666667		
+Speed-2017-485-469	469	zakdoek	handkerchief	0.266666667	0	0.133333333	3	3.066666667	zien	0.474226804	3.416666667	3.5	3.75	1020	HANDKERCHIEF
+Speed-2017-485-470	470	zalm	salmon	0	4.625	3.8125	1.6875	3.6875	proeven	0.334841629	3.5	3.666666667	4	944	SALMON
+Speed-2017-485-471	471	zand	sand	0.2	0	0.466666667	3.866666667	3.133333333	voelen	0.504347826	2.5	3.833333333	4.333333333	671	SAND
+Speed-2017-485-472	472	zee	sea	3.6	2.066666667	3.266666667	2.466666667	4.466666667	zien	0.151260504	3.166666667	5.416666667	5.166666667	1474	SEA
+Speed-2017-485-473	473	zeep	soap	0.066666667	0.6	3.466666667	3.4	3.266666667	ruiken	0.314814815	3.5	4.166666667	5.083333333	788	SOAP
+Speed-2017-485-474	474	zeester	starfish	0.066666667	0	0.466666667	1.6	3.866666667	zien	0.644444444	2.833333333	2.75	4.583333333	3032	STARFISH
+Speed-2017-485-475	475	zeshoek	hexagon	0	0	0	1.333333333	4.266666667	zien	0.761904762	3.583333333	3.916666667	3.666666667		
+Speed-2017-485-476	476	zijde	side	0.333333333	0	0.066666667	3.8	3	voelen	0.527777778	2.666666667	3.333333333	4.333333333	682	SIDE
```

Comment:
There's another think translated as side above.

Reply:
Both can mean "side". However, this concept could also be SILK, especially because "voelen" (feel) is the dominant category.